src/components/CurrencyLogo/index.tsx

```diff
@@ -24,18 +24,21 @@ const StyledNativeLogo = styled(StyledLogo)`
 
 export default function CurrencyLogo({
   currency,
+  symbol,
   size = '24px',
   style,
   ...rest
 }: {
   currency?: Currency | null
+  symbol?: string | null
```

Comment:
I think this can be narrowed to `symbol?: string`, with no `undefined`.
It has no loading state, which I think is what we use `null` to signify.

Reply:
done

Reply:
actually line 505 in TokenRow complains (and the build breaks) if we remove null.